dev/README.md

```diff
@@ -2,7 +2,16 @@
 
 A quick start guide to get Kepler up and running inside your container-based development cluster.
 
-## I just want it built and run it on my cluster
+## To create a new emphemeral local kubernetes cluster
+You can bring the cluster up with:
```

Comment:
just fw to the ![How to use cluster-up](cluster-up/README.md)

Reply:
Right, we redirect